spec/controllers/access_request_controller_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+describe AccessRequestsController, type: :controller do
```

Comment:
we're converting this to a request spec after learning that controller specs are deprecated. :-/